### PR TITLE
added(server): find personalized tutor stats endpoint.

### PR DESCRIPTION
**Pull request by mmoehabb**
ClickUp Task: [https://app.clickup.com/t/86976y333](https://app.clickup.com/t/86976y333)

### CODE REVIEW

**Comment by neuodev** [comment]
🚀 [Nova](https://nova-f36gogx1m-litespace-core.vercel.app) 
🚀 [Luna](https://luna-qzp26brkt-litespace-core.vercel.app) 
🚀 [Nebula](https://nebula-kksd5414n-litespace-core.vercel.app) 
🚀 [Dashboard](https://dashboard-8vhjn8f6i-litespace-core.vercel.app)

File: services/server/src/handlers/user.ts [hunk 1]
```diff
@@ -485,6 +484,52 @@ async function findTutorStats(req: Request, res: Response, next: NextFunction) {
...
+      canceled: false,
+      after: now,
+    }),
+    await lessons.countLessons({
+      users: [id],
+      ratified: true,
+      canceled: false,
+      before: now,
+    }),
+  ]);
```
Comment (mostafakamar2308) [inline services/server/src/handlers/user.ts:519]: executing them in parallel is a nice idea :)

**Review by mostafakamar2308 — APPROVED** [review]

**Comment by neuodev** [comment]
@mmoehabb please rebase.

**Review by neuodev — CHANGES_REQUESTED** [review]

**Comment by neuodev** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>No clickup task found in this pull request</td>
    </tr>
  </tbody>
</table>



<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 79f1b2f98838d3d1016cae2281de6e82b6d60abf
</p>

**Comment by neuodev** [comment]
🚀 [Nova](https://nova-fctz2g5bm-litespace-core.vercel.app) 
🚀 [Luna](https://luna-fti25omdc-litespace-core.vercel.app) 
🚀 [Nebula](https://nebula-hgrl9bus4-litespace-core.vercel.app) 
🚀 [Dashboard](https://dashboard-ow45pwwus-litespace-core.vercel.app)

**Review by neuodev — APPROVED** [review]

**Pull request merged**